<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VarBuildingPay</class>
 <widget class="QDialog" name="VarBuildingPay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="topText">
     <property name="text">
      <string>You still have to pay x items of x types</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="woodLayout">
     <item alignment="Qt::AlignRight">
      <widget class="QLabel" name="woodText">
       <property name="text">
        <string>Wood:</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignLeft">
      <widget class="QSpinBox" name="woodAmount"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="clayLayout">
     <item alignment="Qt::AlignRight">
      <widget class="QLabel" name="clayAsk">
       <property name="text">
        <string>Clay:</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignLeft">
      <widget class="QSpinBox" name="clayAmount"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="stoneLayout">
     <item alignment="Qt::AlignRight">
      <widget class="QLabel" name="stoneAsk">
       <property name="text">
        <string>Stone:</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignLeft">
      <widget class="QSpinBox" name="stoneAmount"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="goldLayout">
     <item alignment="Qt::AlignRight">
      <widget class="QLabel" name="goldAsk">
       <property name="text">
        <string>Gold:</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignLeft">
      <widget class="QSpinBox" name="goldAmount"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="bottomText">
     <property name="text">
      <string>this will give you 0 score</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="okayButton">
       <property name="text">
        <string>Okay</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
